classes/Layer/Layer.js

```diff
@@ -102,12 +102,6 @@ export default class Layer {
         }
     }
 
-    #getPageReference({ id, label, target }) {
```

Comment:
hm. Good catch. I think this should be used, but we'll put it back when that happens.